src/backend/common/models/alliance.py

```diff
@@ -1,5 +1,6 @@
 from typing import List, TypedDict
 
+from backend.common.models.event_team_status import EventTeamStatusPlayoff
```

Comment:
I believe this create a circular import error with `src/backend/common/models/event_team_status.py`

Reply:
The tests are current failing because of this, but I don't want to invest too much time into refactoring this to avoid the circular import until the issue of whether this is the correct data structure (see other comment) is resolved more.